### PR TITLE
Adds send email functionality to cli

### PR DESCRIPTION
**Pull request by aniketsharma00411**
**Note:** This PR contains breaking change for previously implemented down_report() method.

### CODE REVIEW

File: system_monitoring_system/report.py [hunk 1]
```diff
@@ -93,7 +93,6 @@ def send_email(email, password, resource='Main Menu'):
         raise AuthenticationError(msg)
 
     session.quit()
```
Comment (ashok-arora) [inline system_monitoring_system/report.py:97]: Can you make the functions return the filename? It will be helpful for printing it in the GUI.

**Review by ashok-arora — CHANGES_REQUESTED** [review]

**Comment by ashok-arora** [comment]
> **Note:** This PR contains breaking change for previously implemented down_report() method.

Ouch.

File: system_monitoring_system/report.py (hunk 1 above)
Comment (aniketsharma00411) [reply system_monitoring_system/report.py:97]: Which filename?

Comment (ashok-arora) [reply system_monitoring_system/report.py:97]: filename = the name of the report with its location, e.g. `~/Downloads/report_2021-04-23_22-43-41.pdf`

Comment (aniketsharma00411) [reply system_monitoring_system/report.py:97]: Done

**Review by ashok-arora — APPROVED** [review]

**Pull request merged**